README.txt

```diff
@@ -19,16 +19,16 @@ Some libraries Autopsy uses may have different, but similar, open source license
 INSTALLATION
 
 For a Windows installation, all Autopsy dependencies are bundled with the installer provided.
-There is no need for manual installation of additional dependencies if the Windows installer is used.
+Manual installation of additional dependencies are not required if the Windows installer is used.
 
-If you want the Japanese localized version, you must have the Japanese language pack (http://support.microsoft.com/kb/972813) installed and the default locale set to JA. (http://windows.microsoft.com/en-us/windows/change-system-locale#1TC=windows-7).
+If the Japanese localized version is required, the [Japanese language pack](http://support.microsoft.com/kb/972813) must be installed and the default locale must be [set to JA](http://windows.microsoft.com/en-us/windows/change-system-locale#1TC=windows-7).
```

Comment:
This is just a text file designed to be open in notepad or some other simple reader. The brackets won't make a link.